pkg/peer/webrtc_callbacks.go

```diff
@@ -76,8 +75,6 @@ func (p *Peer[ID]) onConnectionStateChanged(state webrtc.PeerConnectionState) {
 	p.logger.Infof("Connection state changed: %v", state)
 
 	switch state {
-	case webrtc.PeerConnectionStateFailed, webrtc.PeerConnectionStateDisconnected, webrtc.PeerConnectionStateClosed:
-		p.sink.Send(LeftTheCall{event.CallHangupUserHangup})
```

Comment:
We should still do it in failed (and presumably closed) so I'd just remove `webrtc.PeerConnectionStateDisconnected` from the middle.